fava_investor/__init__.py

```diff
@@ -44,3 +48,10 @@ def query_func(self, sql):
     def build_tlh_tables(self, begin=None, end=None):
         return libtlh.get_tables(self.query_func, self.config.get('tlh', {}))
 
+    # Balances
+    # -----------------------------------------------------------------------------------------------------------
+    def get_balances(self, interval: Interval):
+        interval_count = 3
```

Comment:
You can simply make this a config parameter with a default value of 3 for now. Or even get rid of the interval functionality for now: showing the latest balance is adequate IMHO. Fava's date functionality allows people to pick an arbitrary date anyway.